### PR TITLE
Apply corrections to managament summary

### PR DESCRIPTION
**Pull request by Venyla**
part of #9

### CODE REVIEW

File: chapters/managementSummary.typ [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 // soll knapp, klar und stark untergliedert sein. (Aus Anleitung Dokumentation FS21 vom SG-I)
 // - Der Umfang beträgt in der Regel 3-5 Seiten.
 // - Bilder sind hier sinnvoll
 
+= Management Summary
+
 The goal of this project was to add new refactorings to the clangd language server to support the use of concepts that were introduced with C++20.
 
-For this, two new refactorings were implemented to be contributed to the LLVM Project.
+Two new refactorings were implemented to contribute to the LLVM Project.
```
Comment (jeremystucki) [inline chapters/managementSummary.typ:11]:
```suggestion
Two new refactoring operations were implemented and the resulting patches have been submitted to the LLVM project.
```

File: chapters/managementSummary.typ [hunk 2]
```diff
@@ -70,30 +71,24 @@ To use clangd in VSCode the extension "clangd" can be used.
...
+Conversely, this circumstance contributes to a slower integration of new changes, given that a majority of contributors are engaged in the project during their leisure hours, impacting the pace of development.
 
-Language servers are a good way to provide language support for more than just one IDE. 
-Having an open source project like LLVM is a excellent idea and is also appreciated by a lot of developers.
-On the other hand, it also makes the process of adding new changes quite slow, as almost all contributors are working on it during their free time.
-
-One of the Pull-Requests got a review from another contributor, who commented that he was looking forward to having the refactoring in clangd, as it could be quite useful.
-This comment seems to conclude the work of this project quite well, and hopefully others will find it a good addition to the language server as well.
-
+One of the Pull-Requests got a review from fellow contributor, who expressed anticipation for the integration of the refactoring in clangd, highlighting its potential usefulness.
```
Comment (jeremystucki) [inline chapters/managementSummary.typ:92]: Check spelling of "pull-requests"

Comment (Venyla) [reply chapters/managementSummary.typ]: han mal de bindestrich usegno

Comment (jeremystucki) [reply chapters/managementSummary.typ]: ja mer setts chli schribe

**Review by jeremystucki — APPROVED** [review]

**Pull request merged**